x-pack/plugins/siem/server/lib/ip_details/elasticsearch_adapter.test.ts

```diff
@@ -49,4 +51,12 @@ describe('elasticsearch_adapter', () => {
       expect(destination).toEqual(mockFormattedDestination);
     });
   });
+
+  describe('#getUsers', () => {
+    test('will format edges correctly', () => {
+      // @ts-ignore Re-work `DatabaseSearchResponse` types as mock ES Response won't match
```

Comment:
Sad to see that ts-ignore :-(

I understand though.

Reply:
Yeah, I wasn't thrilled to add it, but having plumbed a few components front-to-back now, I think I have a few things we can do to drastically improve the dev process. Including better mocks, less fighting with types, and greatly improved maintenance (for when those queries inevitably change! :) Once we fix the `DatabaseSearchResponse` type, then we can just use the entire ES Response as our full mock and then generate the necessary downstream types from that. More on those thoughts later though... :)